src/components/AnonymousChat/ChatDetailHeader.tsx

```diff
@@ -11,17 +11,24 @@ import IcArrowBackWhite from '../../assets/arrow/Ic_arrow_back_white';
 import dimen from '../../utils/dimen';
 import {ANONYMOUS, SIGNED} from '../../hooks/core/constant';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
-import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
+import {COLORS} from '../../utils/theme';
 
 const styles = StyleSheet.create({
   container: {
     display: 'flex',
     flexDirection: 'row',
     height: 50,
+    alignItems: 'center'
+  },
+  bgsigned_primary: {
+    backgroundColor: COLORS.signed_primary
+  },
+  bgBondiBlue: {
+    backgroundColor: COLORS.anon_primary,
     alignItems: 'center',
     borderBottomWidth: 1,
-    borderBottomColor: colors.alto
+    borderBottomColor: COLORS.gray
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code has removed the `borderBottomWidth` and `borderBottomColor` properties from the `container` style and added them to the `bgBondiBlue` style. This could potentially change the appearance of components that use the `container` style but not the `bgBondiBlue` style. If this was intentional and you've checked all uses of these styles, then it's fine. Otherwise, you might want to reconsider this change.



<!-- This is an auto-generated comment by OSS CodeRabbit -->